components/inputs/input-date.js

```diff
@@ -335,6 +343,7 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 		const date = getToday();
 		this._updateValueDispatchEvent(formatDateInISO(date));
 		this._dropdown.close();
+		this.focus();
```

Comment:
I noticed that sometimes when clicking "Set to Today", the dropdown would close but `input-text` would not gain focus (which I think would be the behavior we'd want).